docs/UserGuide.adoc

```diff
@@ -278,21 +332,107 @@ There is no need to save manually.
 
 == Command Summary
 
+=== Table Summary
+Rows highlighted in light blue represent commands that can only be used in the main list and not the replenish list.
+[cols=".<2,.^4,.^4", options="header"]
+
+|===
+|Command |Format |Example
+
+
+|*Help*
+|`help`|
+
+1.2+|*View*
+1.2+|`view\|<list>`
+
+|`view\|main`
+|`view\|replenish`
+
+1.3+|*Search*
+1.3+|`search\|<keyword>[\|<other keywords>]...`
+|`search\|milk\|tea\|pearls`
+|`search\|#drink\|mango pudding`
+|`search\|#fruit #sweet`
+
+
+
+1.2+|{set:cellbgcolor:	#F0F8FF}
+*Add*
+|`add\|<item name>\|<expiry date>`
+|`add\|durian\|30/9/2019`
+
+|`add\|<item name>\|<expiry date>\|<quantity>`
+|`add\|durian\|30/9/2019\|8`
+
+1.2+|{set:cellbgcolor!}
+*Shift*
+|`shift\|<index>`
+|`shift\|2`
+|`shift\|<index>\|<expiry date>\|<quantity>`
+|`shift\|2\|1/1/2020\|2`
+
+
+
+1.3+|{set:cellbgcolor:#F0F8FF}
```

Comment:
Nice use of colour coding to differentiate commands applicable for either list! May I check if the grey cells and white cells are meant to be differentiated?